charts/nri-bundle/README.md

```diff
@@ -177,18 +177,20 @@ honors global options as described below.
 | global.serviceAccount.name | string | `nil` | Change the name of the service account. This is honored if you disable on this chart the creation of the service account so you can use your own |
 | global.tolerations | list | `[]` | Sets pod's tolerations to node taints |
 | global.verboseLog | bool | false | Sets the debug logs to this integration or all integrations if it is set globally |
-| kube-state-metrics.collectors | object | See [`values.yaml`](values.yaml) of the kube-state-metric chart | Collectors configuration of kube-state-metric |
```

Comment:
This does not seem to appear in the current `values.yaml` file, so the `helm docs` command didn't generate it. Let me know if we should add it to `values.yaml`.